Latest changes to iz_ingest

foozleface opened the pull request:
Support for CASIZ_XXX and removed an extra param. Added changes to template.

delarocamcas reviewed (changes requested):
Can you add the term CASIZ_MATCH_SHORT to the iz_config.template.py, otherwise no other changes needed.

foozleface commented:
added support for 5 [or 6] digits alone is a valid CASIZ
CAS [or CASIZ] followed by 3-4 digits is a valid CASIZ

added support for not overwriting exif fields with null optionally

delarocamcas reviewed (approved):
should be good to go. Everything looks consistent
